### PR TITLE
Ks/mapf entries

### PR DESCRIPTION
**Pull request by kstrauch94**
I added a few mapf entries and fixed the ID for the scheduling approach from Jiri (It had the names in it instead of the last names)

### CODE REVIEW

File: krr.bib [hunk 1]
```diff
@@ -1262,6 +1262,20 @@ @inproceedings{ashmor90a
   pages = {1-15}
 }
 
+@inproceedings{aslohaba21a,
+  title = {A New Boolean Encoding for {MAPF} and its Performance with {ASP} and MaxSAT Solvers},
+  author = {R. As{\'i}n Ach{\'a} and R. L{\'o}pez and S. Hagedorn and J. Baier},
+  editor = {Hang Ma and Ivan Serina},
```
Comment (tortinator) [inline krr.bib:1268]:
```suggestion
  editor = {H. Ma and I. Serina},
```
Just initials

File: krr.bib [hunk 2]
```diff
@@ -1262,6 +1262,20 @@ @inproceedings{ashmor90a
...
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  biburl = {https://dblp.org/rec/conf/socs/AchaLHB21.bib},
```
Comment (tortinator) [inline krr.bib:1270]: delete - we do not use this info

Just check the other entrie sin `krr.bib`

File: krr.bib [hunk 3]
```diff
@@ -1262,6 +1262,20 @@ @inproceedings{ashmor90a
...
+  booktitle = {Proceedings of the Fourteenth International Symposium on Combinatorial Search, {SOCS} 2021, Virtual Conference [Jinan, China], July 26-30, 2021},
```
Comment (tortinator) [inline krr.bib:1271]: Please use a crossreference!

File: krr.bib [hunk 4]
```diff
@@ -1262,6 +1262,20 @@ @inproceedings{ashmor90a
...
+  pages = {11-19},
+  publisher = {{AAAI} Press},
```
Comment (tortinator) [inline krr.bib:1273]: chaeck for macro (aka `@string`)

File: krr.bib [hunk 5]
```diff
@@ -1262,6 +1262,20 @@ @inproceedings{ashmor90a
...
+  timestamp = {Wed, 25 Aug 2021 17:11:16 +0200},
+  url = {https://ojs.aaai.org/index.php/SOCS/article/view/18546},
```
Comment (tortinator) [inline krr.bib:1275]: see above

File: krr.bib [hunk 6]
```diff
@@ -5770,6 +5784,20 @@ @inproceedings{capeva05a
...
+  author = {M. Capek and P. Surynek},
+  editor = {Hang Ma and Ivan Serina},
+  bibsource = {dblp computer science bibliography, https://dblp.org},
+  biburl = {https://dblp.org/rec/conf/socs/CapekS21.bib},
+  booktitle = {Proceedings of the Fourteenth International Symposium on Combinatorial Search, {SOCS} 2021, Virtual Conference [Jinan, China], July 26-30, 2021},
+  pages = {153-155},
+  publisher = {{AAAI} Press},
+  timestamp = {Wed, 25 Aug 2021 17:11:16 +0200},
+  url = {https://ojs.aaai.org/index.php/SOCS/article/view/18567},
+  year = {2021}
```
Comment (tortinator) [inline krr.bib:5798]: see above

File: krr.bib [hunk 7]
```diff
@@ -5929,6 +5957,18 @@ @inproceedings{chgose01a
   pages = {408-421}
 }
 
+@article{chhalist2021a,
```
Comment (tortinator) [inline krr.bib:5960]: This is in proceedings, or?

**Review by tortinator — CHANGES_REQUESTED** [review]

File: krr.bib (hunk 7 above)
Comment (tortinator) [reply krr.bib:5946]: Did you change this..?

File: krr.bib [hunk 8]
```diff
@@ -22499,6 +22520,18 @@ @inproceedings{sufestbo18a
   pages = {90-105}
 }
 
+@inproceedings{sulizhkiko20a,
+  title = {Mutex Propagation for {SAT}-based Multi-agent Path Finding},
+  author = {P. Surynek and J. Li and H. Zhang and T. Kumar and S. Koenig},
+  editor = {T. Uchiya and Q. Bai and I. Mars{\'a}{-}Maestre},
+  booktitle = {Proceedings of the Twenty-third International Conference on Principles and Practice of Multi-Agent Systems (PRIMA'20) },
```
Comment (tortinator) [inline krr.bib:22527]: PRIMA might be worth starting with crossrefs

File: krr.bib [hunk 9]
```diff
@@ -22524,6 +22557,16 @@ @inproceedings{surynek10a
   url = {http://www.aaai.org/ocs/index.php/AAAI/AAAI10/paper/view/1768}
 }
 
+@inproceedings{surynek14a,
+  title = {Compact Representations of Cooperative Path-Finding as {SAT} Based on Matchings in Bipartite Graphs},
+  author = {P. Surynek},
+  booktitle = {Proceedings of the Twenty-sixth IEEE International Conference on Tools with Artificial Intelligence (ICTAI'14)},
```
Comment (tortinator) [inline krr.bib:22563]: ICTAI series is already in `procs.bib` - please continue adding the 14 edition

**Review by tortinator — CHANGES_REQUESTED** [review]
Just two minor things - cheers!

File: krr.bib [hunk 10]
```diff
@@ -5929,6 +5943,13 @@ @inproceedings{chgose01a
   pages = {408-421}
 }
 
+@article{chhalist21a,
```
Comment (tortinator) [inline krr.bib:5946]: Must be `inproceedings`

File: procs.bib [hunk 11]
```diff
@@ -2059,6 +2059,13 @@ @proceedings{ictai13
   year = {2013}
 }
 
+@proceedings{ictai14,
```
Comment (tortinator) [inline procs.bib:2062]: Is there an editor?

File: procs.bib [hunk 12]
```diff
@@ -3779,6 +3795,13 @@ @proceedings{socs19
   year = {2019}
 }
 
+@proceedings{socs21,
+  title = {Proceedings of the Fourteenth International Symposium on Combinatorial Search (SOCS'21)},
```
Comment (tortinator) [inline procs.bib:3799]: Add `booktitle`

**Review by tortinator — CHANGES_REQUESTED** [review]
Sorry, still a few issues

File: procs.bib (hunk 11 above)
Comment (kstrauch94) [reply procs.bib:2062]: I can not find an editor for this conference

**Review by tortinator — APPROVED** [review]

**Comment by tortinator** [comment]
Hola, it seems that the checks won't go through - can you fix this? Then we're ready to merge :nerd_face:

**Comment by tortinator** [comment]
Are we now ready to merge..?

**Pull request merged**